apollo-router/tests/snapshots/tracing_tests__traced_basic_composition.snap

```diff
@@ -94,10 +94,6 @@ expression: get_spans()
                 "apollo_private.duration_ns",
                 0
               ],
-              [
-                "apollo_private.http.response_headers",
-                "{}"
-              ],
               [
```

Comment:
Literally don't understand how or why these were changed in the original PR but 🤷 .  Of course, that doesn't block the reversion of it here though. Just mentioning it.